WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -181,6 +182,10 @@ class PostCardStatusViewModel: NSObject {
                 buttons.append(.share)
             }
 
+            if post.status == .publish || post.status == .draft {
```

Comment:
I thought of it but I couldn't find any other condition that should prevent the user from copying the post. I used the same approach [on Android](https://github.com/wordpress-mobile/WordPress-Android/blob/a4dfffe3a4168e88e5c91da4a59adff0c953028b/WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt#L355)

Reply:
For this, I mean the cases that _allow_ the user to duplicate the post. I don't think it's a blocker, especially since we already have matching behavior on Android, and maybe we can consider other cases for a future iteration.

Reply:
> For this, I mean the cases that _allow_ the user to duplicate the post. I don't think it's a blocker, especially since we already have matching behavior on Android, and maybe we can consider other cases for a future iteration.

I agree @mkevins 👍 I'll merge this and reconsider in a next iteration.